ChaosMod/Effects/db/Peds/PedsSpeechController.cpp

```diff
@@ -46,4 +46,27 @@ static void OnTickUnfriendly()
 	}
 }
 
-static RegisterEffect registerEffect2(EFFECT_PEDS_INSULT, nullptr, nullptr, OnTickUnfriendly);
\ No newline at end of file
+static RegisterEffect registerEffect2(EFFECT_PEDS_KIFFLOM, nullptr, nullptr, OnTickUnfriendly);
```

Comment:
this should still be "EFFECT_PEDS_INSULT"

Reply:
oh yeah true, late night in visual makes me do weird things. I'll fix it in the morning